shared/uavobjectdefinition/hwsprf3e.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<xml>
+	<object name="HwSprf3e" singleinstance="true" settings="true" category="HardwareSettings">
+		<description>Selection of optional hardware configurations.</description>
+		<field name="Uart1" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="Disabled">
+			<description/>
+			<options>
+				<option>Disabled</option>
+				<option>DSM</option>
+				<option>SRXL</option>
+				<option>HoTT SUMD</option>
+				<option>HoTT SUMH</option>
+				<option>S.Bus</option>
+				<option>S.Bus Non Inverted</option>
+				<option>IBus</option>
+				<option>ComBridge</option>
+				<option>DebugConsole</option>
+				<option>FrSKY Sensor Hub</option>
+				<option>FrSKY SPort Telemetry</option>
+				<option>GPS</option>
+				<option>HoTT Telemetry</option>
+				<option>I2C</option>
+				<option>LighttelemetryTx</option>
+				<option>MavLinkTX</option>
+				<option>MavLinkTX_GPS_RX</option>
+				<option>MSP</option>
+				<option>OpenLog</option>
+				<option>Storm32Bgc</option>
+				<option>Telemetry</option>
+				<option>TBS Crossfire</option>
+			</options>
+		</field>
+		<field name="Uart2" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="S.Bus">
```

Comment:
XSRF3E has no SDCARD and only 6 PWM-out. I2C is not available on connector. RSSI comes over S.BUS, RSSI-ADC-Pin is not available. The missing features are not implemented on board, but trying to access them will have no influence. Setting up a servo on PWM8 will work, but you could not connect it without soldering the STM-Pin. I found no safe way to detect the correct board. Testing for S.Bus is no option. We could messure VBat. On XSRF3E with USB-Power only you will read 4.xx V. SP Racing F3 Evo gives 0V. But if VBat is connected (1S-Lipo or BEC), detection fails.
I think, the user knows, which board he is setting up and how he has to config the UARTS, ...